test/python/tf_unittest_runner.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+import sys
+import argparse
+import os
+
+#Import ngraph in builds without ngraph embedded
+try:
+    import ngraph
+except:
+    pass
+
+def main():
```

Comment:
Add comment block (using Python style guide) that describes the command line parameters (just brief description) and an example bazel query to generate the list of test targets.